src/bundles/site-logo.less

```diff
@@ -12,11 +12,11 @@
     color: @global-primary-color;
   }
 
-  &.Logo--readonly {
+  &.SiteLogo--readonly {
     cursor: inherit;
   }
 
-  &.Logo--big {
+  &.SiteLogo--normal {
```

Comment:
So then the small one becomes the not normal one by omission right?

Reply:
The educandu `DefaultSiteLogo` supports normal and small, but ELMU and OMA can choose what they overwrite basically. For ELMU there was no case for small, just normal.

Reply:
I also put a comment on the other PR, sorry, I think this would have been the right one. I just don't think things get clearer with this renaming. Now if I don't specify anything, I get a small logo, if I want the `normal` one I have to explicitly say `normal`? Doesn't that somewhat contradict the meaning of `normal`?

Reply:
There's a default set:
```
SiteLogo.defaultProps = {
  readonly: false,
  size: 'normal'
};
```

Reply:
My comment was related to the previous commit (0f52b6a44a0c4dbead3814e05e73dbf6a5cdf608) where there were only SiteLogo and SiteLogo--normal in the CSS. I see that now it is changed, and I like it.